remote_attestation/rust/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "oak_remote_attestation"
+version = "0.1.0"
+authors = ["Ivan Petrov <ivanpetrov@google.com>"]
+edition = "2018"
+license = "Apache-2.0"
+
+[dependencies]
+anyhow = "*"
+env_logger = "*"
+log = "*"
+oak_io = { path = "../../oak_io" }
```

Comment:
Is this dependency really needed? It is bringing `oak_abi`, `oak_derive` and `oak_io` dependencies into Oak Functions which does not sound quite right to me.

Reply:
This is due to using oak_utils to compile the proto files. You should compile the protos using tonic-build directly, as using oak_utils it adds a bunch of Oak dependencies related to handles that we don't need in Oak functions.

Reply:
Done